templates/endpoints_v2/end-to-end-examples/function_calling/serve_llama_3p1.yaml

```diff
@@ -0,0 +1,13 @@
+applications:
+- args:
+    embedding_models: []
+    function_calling_models:
+    - ./llm_configs/meta-llama--Meta-Llama-3_1-8B-Instruct.yaml
+    models: []
+    multiplex_lora_adapters: []
+    multiplex_models: []
+    vllm_base_models: []
+  import_path: aviary_private_endpoints.backend.server.run:router_application
```

Comment:
use the new name space?

Reply:
Good catch, I updated this config to use the new API.